pkg/javadb/client.go

```diff
@@ -142,37 +142,49 @@ func (d *DB) SearchBySHA1(sha1 string) (jar.Properties, error) {
 	}, nil
 }
 
-func (d *DB) SearchByArtifactID(artifactID string) (string, error) {
+func (d *DB) SearchByArtifactID(artifactID, version string) (string, error) {
 	indexes, err := d.driver.SelectIndexesByArtifactIDAndFileType(artifactID, types.JarType)
```

Comment:
What if changing this method to match the version as well as the artifact ID?

Reply:
Do you mean to move the following logic to "trivy-java-db"?
https://github.com/aquasecurity/trivy/blob/2f10ed1c360dd03061481eee97be8c66bd9d5ccc/pkg/javadb/client.go#L157-L188

Or update our sql query in "trivy-java-db"?

Reply:
I meant SQL query. Is it possible?

Reply:
I created https://github.com/aquasecurity/trivy-java-db/pull/23 and updated this PR.
Take a look, when you have time, please.